pkg/aks/components/Config.vue

```diff
@@ -715,6 +729,22 @@ export default defineComponent({
         this.config['logAnalyticsWorkspaceGroup'] = null;
         this.config['logAnalyticsWorkspaceName'] = null;
       }
+    },
+
+    isUserDefinedRouting(neu: boolean) {
```

Comment:
It feels like there's an opportunity to improve the validation interface. What a pain to make this conditional.